sdk/mongocluster/azure-resourcemanager-mongocluster/CHANGELOG.md

```diff
@@ -1,19 +1,24 @@
 # Release History
 
-## 1.1.0-beta.1 (2025-06-04)
+## 1.1.0-beta.1 (2025-07-16)
```

Comment:
The beta version was not auto-incremented - is this expected?
Or is there some other reason this is modifying in place?